packages/fx-core/src/common/tools.ts

```diff
@@ -416,6 +416,10 @@ export function isV3Enabled(): boolean {
   return process.env.TEAMSFX_V3 ? process.env.TEAMSFX_V3 === "true" : true;
 }
 
+export function isDownloadDirectoryEnabled(): boolean {
+  return process.env.DOWNLOAD_DIRECTORY === "true";
```

Comment:
What's the purpose of the feature flag?

Reply:
Since the decision for improving downloading sample is not finalized. I'm considering using this feature-flag to turn it on or off.